### ISSUE TITLE
What versions of Node are pre-compiled?

### ISSUE THREAD

**Issue by aslabs** [tjanczuk/edge#580]
Hello, 
I keep getting the Error "The edge module has not been pre-compiled for node.js version v....".
I tried uninstalling and re-i stalling node with many versions without success.
I don't have the necessary knowledge to compile using visual studio. So what exact version are pre- compiled in the repo?

**Comment by agracio** [comment]
Edge supports NodeJs 4.x, 5.x, 6.x and 7.x.

**Comment by aslabs** [comment]
So how do I resolve the error I get?

**Comment by agracio** [comment]
You haven't provided any info on what you actually doing to get that error. Platform, NodeJs version, steps to reproduce etc...

**Comment by aslabs** [comment]
You are right: sorry.
First, thanks for you help,

I'm actually using node-dde that has edge.js as a  dependency.

After getting the error the first time I noticed they have a reference to a very old version (0.11.0), so I downloaded edge.js from github and forked node-dde project to update the reference. So now I'm using version 7.11.0 (according to the package.json I see in my local node_modules/edge folder).

I had node version 6.11.2 then I tried 7.10.0 because I noticed you have it in the lib/native/win32/x64/ folder.
Then I downgraded to 6.4.0 witch I also saw in that folder.
All without success

What version should I use? Maybe I should first test edge.js alone (without node-dde)??

Thanks again.
